docs/source/analyze/node-resources.md

```diff
@@ -154,3 +154,55 @@ Troubleshoot allows users to analyze nodes that match one or more labels. For ex
           - pass:
               message: This cluster has a node with enough memory and cpu capacity running Mongo Operator.
 ```
+
+### Message Templating
```

Comment:
^ Should this heading be an h2 instead of an h3?

Reply:
Good catch! Thank you!